Makefile

```diff
@@ -227,8 +279,7 @@ release-publish: release-prereqs
 	git push origin $(VERSION)
 
 	# Push images.
-	docker push calico/kube-controllers:$(VERSION)
-	docker push quay.io/calico/kube-controllers:$(VERSION)
+	$(MAKE) push IMAGETAG=$(VERSION) ARCH=$(ARCH)
```

Comment:
I think this is a bit confusing the way this is currently factored, since `make push` will actually re-tag the images that have already been tagged / verified at this point.

Can we split this some way so that it's more clear that the images built and verified above are the same ones that get pushed here? i.e. change `make push` so that it will never build or retag the images?

Reply:
As discussed on the felix PR, I think we want this to fail fast if the image doesn't exist rather than trying to recreate the tag.  Better to fail if anything is ambiguous rather then possibly doing the wrong thing.

Reply:
Yep. Same approach there will be used here. Stay tuned.

Reply:
Updated